### ISSUE TITLE
4 precompute adapters declare grid-axis labels that are not parameters, so those axes can never collapse

### ISSUE THREAD

**Issue by suchethac** [suchethac/tengri#1827]
## Summary

Four precompute adapters declare `AXIS_PARAMS` entries that name no parameter any
`Parameters` can hold. The axes they govern can therefore never be collapsed: the
grid stays at full size, nothing raises, and the SED is unchanged — only larger and
slower to interpolate than the module's own docstring advertises.

This is the residue of the #1738 census. Six of the original ten were near-miss
spellings or prefix drift with an unambiguous live counterpart, and are repaired.
These four are a different cause and a rename cannot close them: the string is an
**internal grid-axis label that was never a user parameter at all**, so there is no
correct name to point at.

## Mechanism

`AXIS_PARAMS` is a tuple of plain strings matched against
`Parameters.get_fixed_values()` keys. Nothing forces the two namespaces to agree, and
the failure mode is inertness rather than an exception — `pname in fixed_values` is
simply always `False`. No warning fires either, because
`DeadPrecomputeAxisWarning` only triggers when *every* declared name is unresolvable
for the model being built.

## The four

Measured by `tests/contract/test_precompute_axis_names_census.py`:

| adapter | axes | dead names |
|---|---|---|
| `nebular/cb19_precompute` | 7 | `log_OH_total`, `log_age_yr_ssp`, `log_U`, `log_nH`, `log_CO`, `dNO`, `HbFrac` |
| `nebular/mappings_photo_precompute` | 4 | `log_age`, `neb_logn` |
| `nebular/cloudy_precompute` | 3 | `log_age` |
| `agn/qsogen_precompute` | 2 | `agn_plslp1`, `agn_ebv` |

`cb19` and the two CLOUDY-family adapters have **every** relevant axis dead, or close
to it. `qsogen` is the mildest case: its grid does still reduce, because it passes
`defaults=_AXIS_DEFAULTS` to `collapse_fixed_axes` for parameters the model never
declares — but a user who declares one of those names free still cannot free the axis,
because the name is not one the model knows.

## `cb19` needs more than a name decision

It also carries **seven axes over a photometry array with six grid dimensions**.
Repairing the names alone would make the collapse contract the filter axis, since
`slice_fixed_axes` indexes positionally (`jnp.tensordot(w, phot, axes=([0], [axis_idx]))`).
The rank check added in 969e43066 will now raise on this rather than silently
contracting the wrong dimension, so the two must be fixed together.

Related: #1737 (cb19_templates.h5 ships as a flat placeholder) — worth resolving
first, since the real grid may settle the axis question.

## Two ways to close each

Per axis, one of:

1. **Declare the parameter.** If the axis is something a user should be able to pin or
   fit (gas density, ionization parameter, C/O), give it a real declaration and point
   `AXIS_PARAMS` at the full prefixed name.
2. **Declare the axis internal.** If it is genuinely not user-facing, say so, and let
   the seam skip it rather than silently trying and failing to match. This wants a
   sentinel or a separate `INTERNAL_AXES` declaration so the census can tell
   "deliberately not collapsible" from "drifted".

Option 2 is probably right for `log_age_yr_ssp` and `log_age` — these look like SSP
grid coordinates rather than free parameters — and option 1 for the nebular physics
axes. But that is a judgement about the models, not something the census can settle.

## Reproducing

```bash
.venv/bin/pytest tests/contract/test_precompute_axis_names_census.py -q
```

The ledger `DEAD_AXIS_NAMES` in that file records the exact set per module and asserts
it in both directions, so repairing a name fails the test and is the prompt to edit
the ledger; a newly broken name fails it too. `test_most_modules_resolve_their_axis_names`
is a ratchet on the healthy count — currently 13 of 17.

## Why no test caught this

The adapter parity suite (`tests/contract/test_precompute_axis_collapse.py`) drives
collapse through a `MagicMock` that answers to whatever name it is handed, and several
tests restated `AXIS_PARAMS` in their own literal rather than reading it from the
adapter. A copy of a declaration cannot disagree with the declaration it copies. Both
patterns are fixed on the #1738 branch; the static census is the complement, since it
needs no template grids and so runs in CI.
